tests/test_grizzly/users/test_messagequeue.py

```diff
@@ -546,6 +546,102 @@ def test_get(self, mq_user: Tuple[MessageQueueUser, GrizzlyContextScenario, Envi
         assert kwargs['exception'] is not None
         request_event_spy.reset_mock()
 
+        # Test queue / expression START
+        response_event_spy.reset_mock()
+
+        the_side_effect = [
+            {
+                'success': True,
+                'worker': '0000-1337',
+                'response_length': 24,
+                'response_time': 1337,
+                'metadata': pymqi.MD().get(),
+                'payload': '',
+            }
+        ]
+
+        # Setup mock to capture json sent to async_messaged
+        class JsonMocker(object):
+            sent_request : Dict[str, Any] = {}
+
+        def mocked_send_json(foo: Any, am_request: Dict[str, Any]) -> None:
+            JsonMocker.sent_request = am_request
+
+        mocker.patch(
```

Comment:
something like this, utilizes pytest-mock plugin and does not require custom implementation.
```
send_json_spy = mocker.patch(
  'grizzly.users.messagequeue.zmq.sugar.socket.Socket.send_json',
  side_effect=[None],
)
...
assert send_json_spy.call_count == 1
args, _ = send_json_spy.call_args_list[0]

assert args[0] == "same as JsonMocker.sent_request"
```